Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -275,6 +276,7 @@ function genericUpdater(instance, state, context) {
       .finally(context.end); // -> end(arraysToBind)
   }
   context.end(); // -> end(generic-updater)
+  return res;
```

Comment:
Should you simply `return Promise.resolve(true)` since you are not doing anything with `res`?

Reply:
Removed the use of res. We should now only return `Promise.resolve(true)`.